Jenkinsfile

```diff
@@ -8,3 +8,4 @@ buildNPM {
   runTestOptions = '--karma.singleRun --karma.browsers ChromeDocker --karma.reporters mocha junit --coverage'
 }
 
+
```

Comment:
This may also have failed if #466 hadn't been merged yet. Thanks for nudging Jenkins though.